requirements.txt

```diff
@@ -1 +1,2 @@
-django-taggit==3.1.0

```

Comment:
Why is this removed here? Add all the other requirements to master and add only Markdown here.